site/en/docs/web-platform/compute-pressure/index.md

```diff
@@ -1,264 +1,221 @@
+
+### PressureObserver
+
+When an `PressureObserver` object is created, it's configured to watch the pressure of supported
+sources, at a given sample rate. The supported sources can be individually observed/unobserved at
+any time during the lifetime of the PressureObserver object. The sample rate cannot be changed
+after the creation of the object.
+
+#### Constructor
+
+`PressureObserver(callback, options)`: Creates a new `PressureObserver` object which will invoke a
+specified callback function when it detects that a change in the values of the source being
+observed has happened.
+
+The constructor takes a [callback](#callback) function and [options](#options)(optional), as parameters.
+
+##### Callback {: #callback }
+
+'callback()': Callback receives the observer handle as well as an array of unread `PressureRecord` objects.
+
+##### Options {: #options }
+
+`PressureObserverOptions`: Contains the sample rate,`sampleRate`, at which the user requests updates.
+
+#### Methods
+
+`PressureObserver.observe(source)`: Tells the 'PressureObserver' which source to observe.
+
+
+`PressureObserver.unobserve(source)`: Tells the 'PressureObserver' to stop observing a source.
+
+
+`PressureObserver.disconnect()`: Tells the 'PressureObserver' to stop observing all sources.
+
+
+`PressureObserver.takeRecords()`: Returns a sequence of [records](#records), since the last callback invocation.
+
 
-### Enabling via about://flags
+`PressureObserver.supportedSources()`(read only): Returns supported source types by the hardware.
 
-To experiment with the API locally, without an origin trial token, enable the
-`#enable-experimental-web-platform-features` flag in about://flags.
+##### Parameters
 
-### Feature detection
+`source`: The source to be observed, for example `cpu`.  This must be one of the [supported source
+types](https://w3c.github.io/compute-pressure/#dfn-supported-source-types).
 
-You should check for the presence of `ComputePressureObserver` in the `window`
-scope to see if the feature is available:
+### PressureRecord
 
-```javascript
-if ('ComputePressureObserver' in window) {
-  // The Compute Pressure API is available.
+The `PressureRecord` interface of the Pressure Observer API describes the pressure trend of a
+source at a specific moment of transition.
+
+#### Instance Properties
+
+`PressureRecord.source` (Read-only): Returns the origin source from which the record is coming.
+
+`PressureRecord.state` (Read-only): Returns the pressure state recorded.
+
+`PressureRecord.time`(Read-only): Returns a high resolution timestamp.
+
+##Feedback {: #feedback }
+
+The Compute Pressure team wants to hear about your experiences using the Compute Pressure API.
+ Please provide your feedback and file issues on [GitHub][issues].
+
+## Examples
+
+### Is the Compute Pressure API supported?
+
+```js
+if ('PressureObserver' in globalThis) {
+  // The Compute Pressure API is supported.
 }
 ```
+### Creating a pressure observer
 
-## Use cases
```

Comment:
I think it would still be valuable to have use cases listed in this doc like we did [there](https://developer.chrome.com/docs/web-platform/document-picture-in-picture/#use-cases)

Reply:
I added the ones we had in the explainer.